phase2/outline
# Run axe-based a11y tests using storybook stories

Utilizing this nice library https://github.com/chanzuckerberg/axe-storybook-testing

How to test -
Open this PR in Gitpod 💸 (there should be a button)
_You can stop the process in the terminal (which runs the default `yarn start`)_
Type in terminal: `yarn storybook.axe`

It will first build Storybook, and then run axe accessibility report on all the stories.

<a href="https://gitpod.io/#https://github.com/phase2/outline/pull/233"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
